quantum/split_common/split_util.c

```diff
@@ -15,8 +15,27 @@
 #    include "rgblight.h"
 #endif
 
+#ifndef SPLIT_USB_TIMEOUT
+#    define SPLIT_USB_TIMEOUT 2500
+#endif
+
 volatile bool isLeftHand = true;
 
+bool waitForUsb(void) {
+    for(uint8_t i = 0; i < (SPLIT_USB_TIMEOUT / 100); i++) {
```

Comment:
```suggestion
    for (uint8_t i = 0; i < (SPLIT_USB_TIMEOUT / 100); i++) {
```

Reply:
Ah this is before future, let me run clang-format over the changes.